extras/porting/linux/supla/control/custom_virtual_relay.h

```diff
@@ -0,0 +1,28 @@
+/*
+ Copyright (C) AC SOFTWARE SP. Z O.O.
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ as published by the Free Software Foundation; either version 2
+ of the License, or (at your option) any later version.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ You should have received a copy of the GNU General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+*/
+
+#ifndef EXTRAS_PORTING_LINUX_SUPLA_CONTROL_CUSTOM_VIRTUAL_RELAY_H_
+#define EXTRAS_PORTING_LINUX_SUPLA_CONTROL_CUSTOM_VIRTUAL_RELAY_H_
+
+#include <supla/control/virtual_relay.h>
+
+namespace Supla {
+namespace Control {
+class CustomVirtualRelay : public virtual Supla::Control::VirtualRelay {};
```

Comment:
why is this class needed?

Reply:
w `custom_relay.h` w definicji `CustomRelay` użyłem 

```
class CustomRelay : public Sensor::SensorParsed<CustomVirtualRelay>,
                   public Template::ControlTemplate<CustomVirtualRelay> {
```
i musiałem zwirtualizować `Supla::Control::VirtualRelay` aby używać np. channel, który występował w dwóch miejscach jak była definicja
```
class CustomRelay : public Sensor::SensorParsed<VirtualRelay>,
                   public Template::ControlTemplate<VirtualRelay> {
```